etcdserver/etcdserverpb/rpc.proto

```diff
@@ -672,6 +672,10 @@ message WatchResponse {
   string cancel_reason = 6;
 
   repeated mvccpb.Event events = 11;
+
+  int64 fragment_count = 7;
+
+  int64 curr_fragment = 8;
```

Comment:
can these be `bool fragment = 7`? it's simpler to have the flag set until the last message so the client only needs to know it should keep accumulating responses